keps/sig-api-machinery/1872-manifest-based-admission-webhooks/README.md

```diff
@@ -155,22 +138,66 @@ plugins:
 ### Reconfiguring manifest file
 
 The manifest file is watched so that the webhook configuration can be
-dynamically changed by editing the contents of the file.
+dynamically changed by editing the contents of the file. In addition to watching
+the file for changes, we would also read the file periodically checking for
+changes.
+
+#### Behavioral details of reconfiguration
+This section details the behavior of the mechanism when encountering non-optimal
+situations.
+
+In cases where:
```

Comment:
and if any of these are true when reading the file on a process start, does the process fail to start? I would expect it to.

Reply:
that matches my expectation, but should be called out explicitly